src/createHeadlessForm.js

```diff
@@ -137,15 +137,16 @@ function buildFieldParameters(name, fieldProperties, required = [], config = {})
  */
 function convertJSONSchemaPropertiesToFieldParameters(
   { properties, required, 'x-jsf-order': order },
-  config = {}
+  config = {},
+  validations
```

Comment:
its removed now and all good here. I want to say this will have a role for the inline rules in the later PR (being a lazy dev and not verified this), let see.